### ISSUE TITLE
[4.0] $gray variables [a11y]

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#33580]
Anything using $gray-600 is getting the color from the bs5 variables.scss

```
$text-muted:   $gray-600 !default;
$form-text-color:  $text-muted !default;
```

This is not a contrast problem when the background color is white but if the background is the blue pattern then the color ratio fails

One place where this is used is the alias description
![image](https://user-images.githubusercontent.com/1296369/117322163-aa8a3600-ae85-11eb-905c-bdea1602477d.png)

I am not sure why but we are overriding the bs5 default colors for the $gray-* variables nor why the $gray-600 is commented out so I am not going to attempt a fix as there must be a reason that I cant see.

@ciar4n @chmst

**Comment by chmst** [comment]
The contrast was sufficient in the former version. It was atum-dark and calculated to fit the contrast ratio requirements. The BS5 form-text is only for white background.

**Comment by brianteeman** [comment]
closed see #34156
